### PR TITLE
Add missing documentation for @GDScript

### PR DESCRIPTION
**Pull request by timoschwarzer**
Adds documentation for:
- `print_debug()`
- `get_stack()`
- `is_instance_valid()` (Someone please check the description for this)

### CODE REVIEW

File: doc/classes/@GDScript.xml [hunk 1]
```diff
@@ -415,6 +415,21 @@
 			<return type="Array">
 			</return>
 			<description>
+				Returns an array of dictionaries representing the current call stack.
+				[codeblock]
+				func _ready():
+					foo()
```
Comment (akien-mga) [inline doc/classes/@GDScript.xml]: Code in code blocks should be indented with 4 spaces.

File: doc/classes/@GDScript.xml [hunk 2]
```diff
@@ -497,6 +512,7 @@
 			<argument index="0" name="instance" type="Object">
 			</argument>
 			<description>
+				Returns True/False whether [code]instance[/code] is a valid object (e.g. has not been deleted from memory).
```
Comment (akien-mga) [inline doc/classes/@GDScript.xml]: "Returns whether" is enough, true/false is implied.

Comment (timoschwarzer) [reply doc/classes/@GDScript.xml]: Took that from the doc above - fixed all occurrences.

File: doc/classes/@GDScript.xml (hunk 1 above)
Comment (timoschwarzer) [reply doc/classes/@GDScript.xml]: Fixed

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**